.travis.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         - LINT=true
     - python: 3.9
       env:
-        - PYTHON=3.7
```

Comment:
Let's move the doc build to 3.8.  It has a long tail of deps and better for it to be stable.